.scripts/overmind/monitor.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Alkemio Server Feedback Aggregator
+#
+# Watches overmind output and aggregates errors to .feedback.log
+# Run in a separate terminal while overmind is active.
+#
+# Usage: ./monitor.sh
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)